models/public/colorization-v2-norebal/colorization-v2-norebal.md

```diff
@@ -0,0 +1,82 @@
+# colorization-v2-norebal
+
+## Use Case and High-Level Description
+
+The `colorization-v2-norebal` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+group of models designed to perform image colorization. For details
+about this family of models, check out the [repository](https://github.com/richzhang/colorization).
+
+The gray-scale or BGR image is first converted to LAB color space. 
+The model input is a blob that consists of a L-channel of image of 1x1x224x224. 
+The mean values need to be subtracted as follows: 50.0
+before passing the image blob into the network. In addition, values must be divided
+by 0.0039.
+
+The model output is a blob that consists of AB channels of image of 1x2x56x56. 
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Colorization  |
+| GFLOPs            | -             |
+| MParams           | -             |
+| Source framework  | Caffe\*       |
+
+## Accuracy
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `data_l`,  shape - `1,1,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is L-channel. 
+Mean values - 50, scale value - 255
+
+### Converted model
+
+Image, name - `data_l`,  shape - `1,1,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is L-channel. 
+Mean values - 50, scale value - 255
```

Comment:
Mean and scale values have been already built in IR model, no need to specify they here.

Explanation: **Input** section specifies how input must be prepared before passing to model.

Reply:
@eizamaliev done

Reply:
I think we misunderstood each other. Mean-scales must present for original input, because user must preprocess image before passing to network. But for converted model user has no needs in such preprocessing, because it already part of the network. So, leave mean-scales in `original` and delete it in `converted`.

Reply:
thank you, already fixed, I sent an erroneous commit